addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -266,19 +266,13 @@
                                             <t t-set="arabic_name" t-value="line.with_context(lang='ar_001').product_id.display_name"/>
                                             <t t-set="english_name" t-value="line.with_context(lang='en_US').product_id.display_name"/>
 
-                                            <span t-field="line.product_id.display_name"
-                                                t-options="{'widget': 'text'}"/>
-
-                                            <t t-if="arabic_name != english_name">
+                                            <t t-if="line.name != english_name and line.name != arabic_name">
```

Comment:
@lordkrandel  Thank you for your review! I updated the commit to be the one as you suggested for stable. As for master, do I create another branch to target master and create  a new PR  for master?

Reply:
You can `@fw-bot up to saas-17.2` and make a new PR for `master`.
See [ref](https://github.com/odoo/odoo/wiki/mergebot#forward-port)
In case, please add a link between the two in the PR description.

Reply:
I don't think I currently have the rights to start the merge process unless you delegate+

Reply:
Hello!  I have some tickets which are getting solved by this PR so I have subscribed to these notifications. You can do `@fw-bot up to 'branch'` regardless of your current rights for merging. It will just tell the forward bot to only create ports up to that specific version upon getting approval 😊